templates/cluster-template.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: ${CLUSTER_NAME}
   labels:
     cni: cilium
+    ccm: linode
```

Comment:
This is for once the CCM is installed via CAAPH (#101) which is needed for k8s 1.29 so the internal/external IP can be set correctly.